lib/core/src/lib/form/components/widgets/core/form.model.ts

```diff
@@ -236,13 +236,17 @@ export class FormModel implements ProcessFormModel {
         for (const field of this.fieldsCache) {
             const variableId = `variables.${field.name}`;
 
-            if (this.isDefined(formValues[variableId]) || this.isDefined(formValues[field.id])) {
+            if (this.canOverrideValue(field, variableId, formValues)) {
```

Comment:
`canOverrideFieldValueWithProcessValue` what do you think? cc @mauriziovitale  @wiktord2000

Reply:
it sounds fine